react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -40,22 +42,30 @@ const SelectorItem: FC<Props> = ({
   imageUrl,
   imageLabel,
   isImpossible,
+  heightImg,
+  widthImg,
 }) => {
   const discount = getDiscount(maxPrice, price)
+
+  const rootClasses = classNames(
+    styles.skuSelectorItem,
+    `${styles.skuSelectorItem}--${slug(variationValue)}`,
+    'relative di pointer flex items-center outline-0',
+    {
+      [styles.skuSelectorItemImage]: isImage,
+      'o-20': isImpossible,
+    }
+  )
+
+  const rootStyles = isImage && heightImg && widthImg ? { height: heightImg, width: widthImg, padding: 0 } : {}
```

Comment:
why `padding: 0`?

Reply:
e o height e width Não deveriam ser passados aqui, mas sim para o <img na linha 89, na minha opinião

Reply:
Because of this: https://github.com/vtex-apps/store-components/blob/master/react/components/SKUSelector/styles.css#L28

Reply:
> e o height e width Não deveriam ser passados aqui, mas sim para o <img na linha 89, na minha opinião

Cada elemento html tem um nome que é o css handle dele e o uso de classnames geralmente vem com alguma lógica javascript, onde vc passa o object das classes com os booleans, colocar isso direto no elemento me parece bagunçar demais as coisas, quando vc poderia simplesmente criar uma variável com o nome do handle do elemento e fazer toda essa lógica em cima, aí no html vc coloca só o que é necessário, o que deixa a estrutura de quais elementos/componentes é usado no componente que vc tá escrevendo mais legível

Reply:
Ahh, eu vi que vc tava falando de colocar direto no img, mas é pq esse elemento é que tava definindo o tamanho do sku, então pra não fazer breaking change eu deixei lá ao invés de direto na imagem